CMakeLists.txt

```diff
@@ -1,15 +1,78 @@
-cmake_minimum_required (VERSION 3.9)
-project (newuoa VERSION 0.1.0 DESCRIPTION "libnewuoa")
+cmake_minimum_required(VERSION 3.16)
+
+project(newuoa VERSION 0.1.1 DESCRIPTION "C++ implementation of NEWUOA algorithm with C interface")
```

Comment:
It worth to change version to 0.1.2 because there is already a tag for 0.1.1 https://github.com/elsid/newuoa-cpp/releases .